src/odemis/gui/cont/tabs.py

```diff
@@ -3202,6 +3204,7 @@ def __init__(self, name, button, panel, main_frame, main_data):
         self._spot_stream = spot_stream
 
         self._ccd_stream = None
+        self._ccd_spe = None
```

Comment:
What does SPE stand for?

Reply:
Good question! This is a stream controller for the ccd stream, it's initialized in here 

```
self._ccd_spe = self._stream_controller.addStream(ccd_stream,
                                add_to_view=self.panel.vp_align_lens.view)
```

Maybe @pieleric can tell us, I used it as it's defined.

Reply:
Maybe it is nice to add a comment with the explanation behind the initialization of this variable?

Reply:
Stream Panel Entry.

Reply:
Added a comment for the reason of keeping it as attribute